tests/terratest/functions/cleanupConfigTF.go

```diff
@@ -0,0 +1,24 @@
+package functions
+
+import (
+	"fmt"
+	"os"
+)
+
+func CleanupConfigTF(module string) {
+
+	f, err := os.Create("../../modules/" + module + "/main.tf")
+
+	if err != nil {
+		fmt.Println(err)
```

Comment:
Don't print out. If you don't want to return an. error log it instead then. Printing to the command line is not very useful. So either return the error or log the statement. But I would like an explanation first why you aren't returning the error.

Reply:
Returning the error is ideal.  I will update this @igomez06 👍